### DISCUSSION TITLE
Table panel : Column filter unusable if the panel doesn't have a title.

### DISCUSSION THREAD

**Discussion by Irrational-NX** [grafana/grafana#48610] [Dashboards, Panels & Visualizations]
Hello everyone, we're trying to do some very compact dashboards here. And we're having problems with the relation between the column filters and the panel title (or absence of...). 

**What happened**:
Not all of our panels needs a title, so we're removing that to gain a little bit of space, but for the "table" panels, then we can't use the column filters. When overing the mouse on top of the columns, we get the options for the panel (view, edit, share...)

**What you expected to happen**:
Filters icon should be clickable and usable.

**How to reproduce it (as minimally and precisely as possible)**:
Create a table panel, fill it with a query, activate column filters and remove panel title.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.5.1 (pretty sure it's not exclusive to this version though)
- Data source type & version: -
- OS Grafana is installed on: -
- User OS & Browser: Windows10/Chromium
- Grafana plugins: -
- Others: -

**Comment by torkelo** [comment]
It's currently a limitation for panels without title, that the top area cannot be interacted with as it will be used for dragging the panel around and opening the panel menu. 

We have a planned solution for this, but not yet implemented:
https://github.com/grafana/grafana/issues/31080

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
